BrackeysBot.cs

```diff
@@ -21,14 +22,17 @@ public sealed class BrackeysBot
         public static IConfiguration Configuration { get; set; }
         public const int Version = 3;
 
+        public bool OriginalProcess = true;
+
         public DataModule Data { get; set; }
         public CommandHandler Commands { get; set; }
 
         private IServiceProvider _services;
-        private DiscordSocketClient _client;
+        public DiscordSocketClient _client;
```

Comment:
No need to make this public. Also you made it public but now has the wrong naming style.